packages/blade/src/tokens/global/fontFamily/fontFamily.native.ts

```diff
@@ -2,6 +2,7 @@ import { Platform } from 'react-native';
 import type { FontFamily } from './types';
 
 export const fontFamily: FontFamily = {
-  text: 'Lato',
+  text: 'Inter',
+  heading: 'Tasa Orbiter',
```

Comment:
are there any additional installation steps that we need to add or they remain same as old since people were already installing lato?

Reply:
I haven't changed documentation yet. Will change in the end. Installation steps will change a bit from fontsource to blade/fonts.css

Reply:
can you make a note of it in your TODO so we don't forget? or maybe an umbrella issue that tracks all these tiny things?